packages/wallet-core/modules/wallet/index.js

```diff
@@ -0,0 +1,7 @@
+import reducers from './reducers';
```

Comment:
we follow the following structure of redux modules:
```
export const walletTypes = {
    WALLET_LOAD: 'wallet/LOAD'
    WALLET_FUNDS_ADD: 'wallet/funds/ADD'
}

export const walletActions = {
   loadWalletAction: payload => ({type: walletTypes.WALLET_LOAD, payload})
}

export const walletOperations = {
  ...walletActions, 
   loadWalletFromKeystoreOperation: path => async (dispatch, getState) => {
        ...complex operation here
   }, 

   export const walletReducers  = {
        addWalletFundsReducer: (state, action) => {}
        ...
   }

   export const walletReducer = (state, action) => {
       //main reducer
       switch(action.type){ ... }
   }
}

 export default walletReducer;

Reply:
this allows us to build testable redux ducks

if the duck grows to be too complex, we can separate it to different files (for actions, operations, reducers...)

Reply:
I've applied the changes max. Please review